scripts/exdump_monitor.sh

```diff
@@ -194,7 +194,7 @@ DTIM_earliest_atmsdb=-1.50
 DTIM_latest_atmsdb=-0.51
 
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 adpupa osbuv8 1bmhs \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 adpupa osbuv8 1bmhs \
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.